aspnetcore/migration/22-to-30.md

```diff
@@ -1025,7 +1025,13 @@ The following code shows the ASP.NET Core 2.2 template-generated `Program` class
 
 <xref:Microsoft.AspNetCore.Hosting.IWebHostBuilder> remains in 3.0 and is the type of the `webBuilder` seen in the preceding code sample. <xref:Microsoft.AspNetCore.Hosting.WebHostBuilder> will be deprecated in a future release and replaced by `HostBuilder`.
 
-The most significant change from `WebHostBuilder` to `HostBuilder` is in [dependency injection (DI)](xref:fundamentals/dependency-injection). When using `HostBuilder`, you can only inject <xref:Microsoft.Extensions.Configuration.IConfiguration> and <xref:Microsoft.AspNetCore.Hosting.IHostingEnvironment> into `Startup`'s constructor. The `HostBuilder` DI constraints:
+The most significant change from `WebHostBuilder` to `HostBuilder` is in [dependency injection (DI)](xref:fundamentals/dependency-injection). When using `HostBuilder`, you can only inject the following into `Startup`'s constructor:
+
+* <xref:Microsoft.Extensions.Configuration.IConfiguration>
+* <xref:Microsoft.AspNetCore.Hosting.IHostingEnvironment>
```

Comment:
The announcement has this one as `IHostEnvironment`. `IHostingEnvironment` is obsolete as of 3.0.

Reply:
ooops! Good :eye: @serpent5!

Reply:
I'm going to 1/2 take the hit on that one .... **_and 1/2 blame Azure AD!!!!!_** 😠 :smile: